src/colour/social/index.md

```diff
@@ -57,53 +59,55 @@ Our social palette has:
 
 ## Use colour to reflect tone
 
-Our updated palette has been developed to allow a range in expression across the inform to inspire scale. There are moments where the brand needs to feel functional and serious, guiding users seamlessly to the content and services they need. With the introduction of new channels such as social, there are also moments where the brand needs impact and visual differentiation.
+Our updated palette has been developed to allow a range in expression across the inform to inspire scale. 
+
+There are moments where the brand needs to feel functional and serious, guiding users seamlessly to the content and services they need. With the introduction of new channels such as social, there are also moments where the brand needs impact and visual differentiation.
 
-Within our palette there are two approaches to colour application; tonal colours and companion colours.
+Within our palette there are two approaches to colour application: tonal colours and companion colours.
 
 Depending on tonal requirement, each can be used to achieve a different level of expression. The following guidance details the use of these two approaches.
 
 <!-- Obviously, that's not the right heading, needs proper design -->
 
 ### Inform <--------> Inspire
 
-![TODO](./tone-black-and-white.png)
+![Title graphic for "National mourning guidance" in black and white.](./tone-black-and-white.png)
 B&W
 
-![TODO](./tone-dark-shade.png)
+![Title graphic for "Get support with the cost of living" in a dark shade of blue.](./tone-dark-shade.png)
 Dark shade
 
-![TODO](./tone-primary-blue.png)
+![Title graphic for "Fuel duty will be frozen next year" in primary blue colour.](./tone-primary-blue.png)
 Primary blue
 
-![TODO](./tone-tonal-colour.png)
+![Title graphic for "Get help with your pension" in primary blue and light blue tonal colour.](./tone-tonal-colour.png)
 Tonal colour
 
-![TODO](./tone-companion-colour.png)
+![Title graphic for "Learning to drive a car" in companion colours of accent green and dark blue shade.](./tone-companion-colour.png)
 Companion colour
 
 ### Tonal colour examples
 
-{% callout %}
-Indicative examples for illustrative purposes only.
-{% endcallout %}
-
 Colour can be used to reflect tone of a message.
 
 For more sensitive messaging, colours from within the same tonal range are used to feel more serious, informative and functional.
 
-![TODO](./example-tonal-1.png)
+{% callout %}
+Indicative examples for illustrative purposes only.
+{% endcallout %}
+
+![Title graphic for "Fuel duty will be frozen next year" in green tonal colours combination.](./example-tonal-1.png)
```

Comment:
Below you always write "colour combination" but here "colours combination". I assume the additional "s" here is a typo?